### PR TITLE
AC-267 Edit forms functionality

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**
This only works so far for editing Vitals from PatientsDashboardActivity(screenshot attached). I figure out that this should be done from ActiveVisits activity later, so I will add this tomorrow.

@rkorytkowski @tmarzeion please review this PR

 
screenshot_1483019525

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/vitals/PatientDashboardVitalsPresenter.java [hunk 1]
```diff
@@ -16,28 +16,60 @@
...
+    private void loadVitalsFromServer() {
+        if (NetworkUtils.isOnline()) {
+            new VisitApi().syncLastVitals(mPatient.getUuid(), new DefaultResponseCallbackListener() {
+                @Override
+                public void onResponse() {
+                    loadVitalsFromDB();
+                }
+
+                @Override
+                public void onErrorResponse() {
```
Comment (tmarzeion) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/vitals/PatientDashboardVitalsPresenter.java]: @AdamGrzybkowski This needs to be dealed with :sunglasses:

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/formdisplay/FormDisplayMainPresenter.java [hunk 2]
```diff
@@ -133,4 +137,11 @@ public void addFragment(Fragment fragment) {
         }
         return ret;
     }
+
+    public static FormResource getFormResource(String name) {
```
Comment (AdamGrzybkowski) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/formdisplay/FormDisplayMainPresenter.java]: @tmarzeion Is this really the link you wanted to paste here ? :)

Comment (tmarzeion) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/formdisplay/FormDisplayMainPresenter.java]: @AdamGrzybkowski [Yes](https://www.youtube.com/watch?v=y6120QOlsfU) :)

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/vitals/PatientDashboardVitalsPresenter.java (hunk 1 above)
Comment (AdamGrzybkowski) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/vitals/PatientDashboardVitalsPresenter.java]: @tmarzeion done

**Pull request merged**